.travis.yml

```diff
@@ -29,20 +29,18 @@ before_install:
   - bash ci/scripts/conda_setup.sh
   - export PATH="$HOME/miniconda/bin:$PATH"
 
-  - pip install git+https://github.com/dask/distributed.git --upgrade
-
   # Start containers
   - export DOCKER_IP=127.0.0.1
   - docker-compose up -d
   - docker ps -a
   - docker images
 
 install:
-  - which python
-  - python setup.py install
+  - conda install distributed
```

Comment:
By default conda only subscribes to the `default` channel.  I recommend adding `-c conda-forge`